doc/requirements.txt

```diff
@@ -0,0 +1,4 @@
+furo
+pyyaml
```

Comment:
Isn't yaml now part of Python 3.11?

Reply:
Might be. Yet, some of the images are based on Debian Bullseye, where Python 3.9 is the default. Hence, let's better keep it until the stable Debian ships with Python >=3.11.

Reply:
ok